src/components/__stories__/login_header.tsx

```diff
@@ -5,5 +5,5 @@ import LoginHeader from "../login_header"
 
 storiesOf("LoginHeader", LoginHeader)
   .add("Simple login header", () => (
-    <LoginHeader/>
+    <LoginHeader title={"Welcome back, please log in to your account"}/>
```

Comment:
Just to save you googling, props like this are accessed via `this.props.children` as opposed to `this.props.title`.